.github/workflows/ci.yml

```diff
@@ -67,20 +70,14 @@ jobs:
       - name: Install dependencies
         run: npm install
 
+      - name: Build site
+        run: npm run build
+
       - name: Install Playwright
         run: npx playwright install
 
-      - name: Wait for the Netlify Preview
-        uses: kamranayub/wait-for-netlify-action@v2.1.0
-        id: waitForNetlify
-        with:
-          site_name: 'niaefeup-frontend-storybook'
-          max_timeout: 300 # 5 minutes
-        env:
-          NETLIFY_TOKEN: ${{ secrets.NETLIFY_TOKEN }}
-
       - name: Test
-        run: npm run test-storybook:coverage -- --url ${{ steps.waitForNetlify.outputs.url }} --ci
+        run: npm run test-storybook:ci
```

Comment:
I'm curious as to how this CI works :-)

Reply:
https://github.com/storybookjs/test-runner?tab=readme-ov-file#2-running-against-locally-built-storybooks-in-ci

TLDR: it builds the storybook locally, and serves it under a local HTTP server on the CI and then tests it. I don't know why it's hanging indefinitely, yet :sweat_smile: